jupyter-notebooks/analysis-ready-data/ard_2_use_case_1.ipynb

```diff
@@ -98,17 +138,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 33,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# if your Planet API Key is not set as an environment variable, you can paste it below\n",
+    "\"\"\"\n",
+    "If your Planet API Key is not set as an environment variable, you can paste it below\n",
+    "Note: please be sure to follow the security guidelines put forth by your \n",
+    "organization when using this API Key in the keychain\n",
+    "\"\"\"\n",
     "API_KEY = os.environ.get('PL_API_KEY', 'PASTE_YOUR_KEY_HERE')\n",
```

Comment:
Let's revisit this Auth piece. I'm working on getting details for best practices related to the [keyring module](https://pypi.org/project/keyring/) we may start using across all our notebooks.